source/overloading.tex

```diff
@@ -3599,34 +3588,37 @@
 a relational operator function, or
 a three-way comparison operator function.
 
-\rSec2[over.ass]{Assignment}
+\rSec2[over.ass]{Simple assignment}
```

Comment:
I think we should indent this to `\rSec3` since this is a kind of binary operator function.

Reply:
A hanging paragraph! Yay!

Reply:
Fixed.